package.json

```diff
@@ -28,10 +28,15 @@
     "posttest": "nyc report --reporter=json && codecov -f coverage/*.json",
     "typings": "typings install"
   },
-  "dependencies": {},
+  "dependencies": {
+    "bluebird": "^3.5.0",
```

Comment:
Because this is an SDK, I'd like us to keep our dependencies to an absolute minimum - is this really necessary? Keep in mind, this means everyone who uses this library inherits this.